apps/www/components/PoliticsCommunityQuestionnaire/config.ts

```diff
@@ -0,0 +1,67 @@
+import { scaleOrdinal } from 'd3-scale'
+
+// TODO: use correct color palette
+export const QUESTIONNAIRE_BG_COLOR = '#ffffff'
+const COLORS = ['#fff75b', '#ffe003']
+
+export const questionColor = scaleOrdinal(
+  [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+  COLORS,
+)
+
+type QuestionConfigType = {
+  ids: number[]
+  valueLength?: {
+    min?: number
+    max?: number
+  }
+  hint?: string
+}
+
+// TODO: adjust min and max length
+export const QUESTIONS: QuestionConfigType[] = [
+  { ids: [0], valueLength: { min: 0, max: 50 } },
+  { ids: [1], valueLength: { min: 0, max: 150 } },
+  {
+    ids: [2],
+    valueLength: { min: 0, max: 150 },
+    hint: 'Tippen Sie eine Antwort an, um den ganzen Fragebogen dieser Person zu sehen.',
+  },
+  { ids: [3], valueLength: { min: 0, max: 150 } },
+  { ids: [4], valueLength: { min: 0, max: 150 } },
+  { ids: [5, 6], valueLength: { min: 0, max: 100 } },
+  {
+    ids: [7],
+    valueLength: { min: 0, max: 200 },
+  },
+  { ids: [8], valueLength: { min: 0, max: 200 } },
+  { ids: [9], valueLength: { min: 0, max: 200 } },
+  { ids: [10] },
+  { ids: [11], valueLength: { min: 0, max: 200 } },
+  { ids: [12], valueLength: { min: 0, max: 200 } },
+  { ids: [13], valueLength: { min: 0, max: 100 } },
+  { ids: [14], valueLength: { min: 0, max: 100 } },
+  { ids: [15], valueLength: { min: 0, max: 100 } },
+  { ids: [16], valueLength: { min: 0, max: 100 } },
+  { ids: [17], valueLength: { min: 0, max: 100 } },
+  { ids: [18], valueLength: { min: 0, max: 100 } },
+  { ids: [19], valueLength: { min: 0, max: 100 } },
+  { ids: [20] },
+  { ids: [21], valueLength: { min: 0, max: 100 } },
+  { ids: [22], valueLength: { min: 0, max: 100 } },
+  { ids: [23] },
+  { ids: [24] },
+  { ids: [25], valueLength: { min: 0, max: 100 } },
+]
+
+export const EDIT_QUESTIONNAIRE_PATH = '/2023/07/12/politikfragebogen-community'
+export const OVERVIEW_QUESTIONNAIRE_PATH =
+  '/26-fragen-zur-politik-ihre-antworten'
+
+export const PERSON_PAGE_PATH = 'politikfragebogen-community'
+export const PERSON_SHARE_TEXT = '26 Fragen zur Politik – die Antworten von '
+
+export const QUESTIONNAIRE_SLUG = 'politiker-fragebogen-community'
```

Comment:
A bit confusing that this is "politiker-fragebogen" and "politikfragebogen" elsewhere?

Reply:
yes, i fucked up the slug. But Sharon is going to kill me, if I tell her that i need another slug